deployment/E_heroku.md

```diff
@@ -1 +1,277 @@
-## FPO
+### What we'll need
+
+- a working application
+
+### Goals
+
+Our main goal for this is guide is to get a Phoenix application running on Heroku.
+
+## Steps
+
+- Sign up for Heroku and install the Toolbelt
+- Create the Heroku application
+- Add the Phoenix static buildpack
+- Make our project Heroku-ready
+- Deploy!
+
+## Sign up for Heroku and install the Toolbelt
+
+Signing up to Heroku is very simple, just head over to [https://signup.heroku.com/www-header](https://signup.heroku.com/www-header).
```

Comment:
Is there anything users might care to know about the signup process - different tiers/plans, any random bits? If you feel like there's something to say, I don't think we need to actually say much, maybe just a sentence or two.